docs/UserGuide.md

```diff
@@ -150,22 +150,24 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 * This command will check for duplicate phone number before edit is executed. If the new phone number is used by another person, the command will not be executed.
 
 Examples:
-*  `edit 3 n/Bobby Tay e/bobbytay@u.nus.edu` Edits contact displayed at index 3 and changes the name of the contact to `Bobby Tay` and email to `bobbytay@u.nus.edu` respectively.
-*  `edit 1 t/PRUActive Saver III t/PRUActive Cash` Edits contact displayed at index 1 to change its tags to only `PRUActive Saver III` and `PRUActive Cash`. It will remove all other previous tags that are previously associated with the contact.
+* `edit 3 n/Bobby Tay e/bobbytay@u.nus.edu` Edits contact displayed at index 3 and changes the name of the contact to `Bobby Tay` and email to `bobbytay@u.nus.edu` respectively.
+* `edit 1 t/PRUActive Saver III t/PRUActive Cash` Edits contact displayed at index 1 to change its tags to only `PRUActive Saver III` and `PRUActive Cash`. It will remove all other previous tags that are previously associated with the contact.
 
 ### Locating persons by person's information: `find`
 
-Finds persons based on a category of a person's information, e.g. name, email, phone number, or tags.
+Finds persons using specified keywords for a specified category of a person's information, e.g. either name, email, phone number, or tags.
 
-Format: `find n/NAME|e/EMAIL|p/PHONE|t/TAG`
+Format: `find n/NAME|e/EMAIL|p/PHONE|t/TAG...`
```

Comment:
Would it be better to specify everything instead of using `...` ?

Reply:
The `...` indicates the option that there can be multiple tag parameters supplied by the user, following the definition of `...` that is stated at [line 54](https://github.com/AY2324S2-CS2103-F15-1/tp/commit/dfdc78e04f9d24d5c4a5cbb5ada84484f78f6c6b#diff-b50feaf9240709b6b02fb9584696b012c2a69feeba89e409952cc2f401f373fbR54) of this file.

Reply:
Ah missed out on that, LGTM then